content/html/concepts/tags/terms/dl/dl.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '<dl>'
+Description: 'Dislpays terms and details, commonly for metadata purposes.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Lists'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<dl>` tag is a description list that displays terms and details, commonly for metadata purposes.
+
+## Syntax
+
+```html
+<dl>
+  <!-- <dt> terms here -->
+  <!-- <dd> details here -->
+</dl>
+```
+
+Inside the `<dl>` tag, one or more `<dt>` and `<dd>` tags can be used.
+
+**Note**: For Apple VoiceOver, the `<dl>` tag will be recognized and announced as a "list" on macOS devices whereas it [will not](https://cdpn.io/pen/debug/NzGaKP) be recognized on iOS devices.
+
+With iOS, the `<dl>` be recognized by may not always be the best option
```

Comment:
There seems to be some missing words in this sentence.

Reply:
I think I was rewriting the previous sentence on line # 28 by accident. 😌 I removed the text on line # 30.